modules/core/04-channel/keeper/handshake.go

```diff
@@ -98,51 +98,19 @@ func (k Keeper) ChanOpenTry(
 	ctx sdk.Context,
 	order types.Order,
 	connectionHops []string,
-	portID,
-	previousChannelID string,
+	portID string,
 	portCap *capabilitytypes.Capability,
 	counterparty types.Counterparty,
 	counterpartyVersion string,
 	proofInit []byte,
 	proofHeight exported.Height,
 ) (string, *capabilitytypes.Capability, error) {
 	var (
-		previousChannel      types.Channel
-		previousChannelFound bool
+		channelID string
 	)
 
-	channelID := previousChannelID
-
-	// connection hops only supports a single connection
-	if len(connectionHops) != 1 {
```

Comment:
Shouldn't we keep this check?

Reply:
yes, we need should keep this check. I'll do it in new commit.